Next

runleonarun opened the pull request:
## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If linked to an open
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
<!--
Uncomment if you're publishing docs for a prerelease version of dbt (delete if not applicable):
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
-->
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

beericht committed:
Update cloud-ci-job.md

Typo
seleted --> selected

mirnawong1 committed:
Merge branch 'next' into patch-1

mirnawong1 committed:
Merge pull request #2200 from beericht/patch-1

Update cloud-ci-job.md

runleonarun committed:
merging current

runleonarun committed:
minor change

runleonarun committed:
Merge branch 'current' into next

matthewshaver committed:
update next branch

matthewshaver committed:
New dbt cloud IA

matthewshaver committed:
Merge branch 'next' into cloud-ia

matthewshaver committed:
Update _redirects

matthewshaver committed:
Update _redirects

matthewshaver committed:
Updates to embedded markdown links

matthewshaver committed:
Merge branch 'cloud-ia' of https://github.com/dbt-labs/docs.getdbt.com into cloud-ia

matthewshaver committed:
Update website/docs/docs/cloud/git/connect-gitlab.md

matthewshaver committed:
Update audit-log.md

matthewshaver committed:
Update set-up-sso-azure-active-directory.md

matthewshaver committed:
Update set-up-sso-google-workspace.md

matthewshaver committed:
Update set-up-sso-okta.md

matthewshaver committed:
Apply suggestions from code review

matthewshaver committed:
Update website/docs/docs/cloud/manage-access/set-up-sso-saml-2.0.md

matthewshaver committed:
Apply suggestions from code review

matthewshaver committed:
Update website/docs/docs/cloud/manage-access/set-up-bigquery-oauth.md

matthewshaver committed:
Update set-up-bigquery-oauth.md

matthewshaver committed:
Update connect-your-database.md

matthewshaver committed:
Update setup-azure.md

matthewshaver committed:
Update set-up-bigquery-oauth.md

matthewshaver committed:
Update set-up-sso-azure-active-directory.md

matthewshaver committed:
Update set-up-sso-google-workspace.md

matthewshaver committed:
Update website/docs/docs/cloud/manage-access/set-up-sso-google-workspace.md

matthewshaver committed:
Apply suggestions from code review

matthewshaver committed:
Update website/docs/docs/cloud/manage-access/set-up-sso-google-workspace.md

matthewshaver committed:
Update website/docs/docs/cloud/git/connect-gitlab.md

matthewshaver committed:
Update set-up-sso-okta.md

matthewshaver committed:
Update set-up-sso-okta.md

matthewshaver committed:
Update set-up-sso-azure-active-directory.md

matthewshaver committed:
Update website/docs/docs/cloud/git/setup-azure.md

matthewshaver committed:
Update set-up-sso-okta.md

matthewshaver committed:
Update setup-azure.md

matthewshaver committed:
Update set-up-sso-okta.md

matthewshaver committed:
Update deployment-overview.md

matthewshaver committed:
Update deployment-overview.md

matthewshaver committed:
Update deployment-overview.md

matthewshaver committed:
Merge pull request #2932 from dbt-labs/cloud-ia

New dbt cloud IA

matthewshaver committed:
Changing sidebar arrangement for new items

matthewshaver committed:
New additions to next

matthewshaver committed:
Update architecture.md

matthewshaver committed:
updating pdf filename

matthewshaver committed:
Merge branch 'more-next' of https://github.com/dbt-labs/docs.getdbt.com into more-next

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Merge branch 'current' into next

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Moving assets folder

matthewshaver committed:
Merge branch 'more-next' of https://github.com/dbt-labs/docs.getdbt.com into more-next

matthewshaver committed:
removing dead link

mirnawong1 committed:
Update develop-in-the-cloud.md

updating ide docs to include some features that affect development. see [slack thread](https://dbt-labs.slack.com/archives/C043H8Z0P7H/p1678724314676849?thread_ts=1678122006.588049&cid=C043H8Z0P7H)

added:
- individual git commit/revert files
- yaml autocomplete
- 2+models

Commit:
ide updates

Commit:
updates

mirnawong1 committed:
Merge branch 'current' into mwong-ide-tweaks

mirnawong1 committed:
Update develop-in-the-cloud.md

matthewshaver committed:
Merge branch 'next' into more-next

matthewshaver committed:
Update sidebars.js

matthewshaver committed:
Update sidebars.js

matthewshaver committed:
Fixing broken links

matthewshaver committed:
Merge branch 'more-next' of https://github.com/dbt-labs/docs.getdbt.com into more-next

matthewshaver committed:
Update website/docs/docs/cloud/about-cloud/architecture.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

matthewshaver committed:
Update website/docs/docs/cloud/about-cloud/about-cloud-ide.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
Update architecture.md

matthewshaver committed:
New additions to next (#2983)

## What are you changing in this pull request and why?
More additions to next for IA moves

## Checklist
<!--
Uncomment if you're publishing docs for a prerelease version of dbt
(delete if not applicable):
- [ ] Add versioning components, as described in [Versioning
Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
-->
- [ ] Review the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
and [About
versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this
PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link
testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally)
to update the links that point to the deleted page

matthewshaver committed:
Adding quickstarts to sidebar

matthewshaver committed:
Updating Core docs locations

matthewshaver committed:
Fixing links

matthewshaver committed:
Fixing broken links

matthewshaver committed:
More link fixes

matthewshaver committed:
Link fixes again

matthewshaver committed:
Changes based on Sai's feedback

matthewshaver committed:
Adding info to Architecture

Commit:
Merge branch 'origin/next' into mwong-ide-tweaks

mirnawong1 committed:
Update develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

mirnawong1 committed:
Update website/docs/faqs/API/rotate-token.md

mirnawong1 committed:
Update develop-in-the-cloud.md

matthewshaver committed:
Merge branch 'current' into next

matthewshaver committed:
Update audit-log.md

matthewshaver committed:
Fixing link

matthewshaver committed:
Merge branch 'next' of https://github.com/dbt-labs/docs.getdbt.com into next

runleonarun committed:
Merge branch 'next' into mwong-ide-tweaks

mirnawong1 committed:
Update website/docs/docs/cloud/develop-in-the-cloud.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

matthewshaver committed:
updates to Architecture based on feedback

matthewshaver committed:
More feedback

mirnawong1 commented on website/docs/docs/cloud/about-cloud/architecture.md:
```diff
@@ -0,0 +1,68 @@
+description: "Information about the architecture, communication, and security of dbt Cloud"
+---
+
+This page is intended to help practitioners and those interested in the architecture and data flow of the hosted dbt Cloud product.
```
```suggestion
This page helps practitioners and those interested in dbt Cloud's architecture and data flow.
```

mirnawong1 commented on website/docs/docs/cloud/about-cloud/architecture.md:
```diff
@@ -0,0 +1,68 @@
+
+## About dbt Cloud architecture
+
+The dbt Cloud application comprises a set of static and dynamic components. The static components are constantly running to serve highly available dbt Cloud functionality, for example, the dbt Cloud web application. The dynamic components are created ad-hoc to fill background jobs or requests to use the IDE. 
```
```suggestion
The dbt Cloud application has two types of components: static and dynamic. The static components are always running to serve highly available dbt Cloud functions, like the dbt Cloud web application. On the other hand, the dynamic components are created ad-hoc to handle tasks such as background jobs or requests to use the IDE.
```

mirnawong1 commented on website/docs/docs/cloud/about-cloud/architecture.md:
```diff
@@ -0,0 +1,68 @@
+
+dbt Cloud uses PostgreSQL for its backend, S3-compatible Object Storage systems for logs and artifacts, and a Kubernetes storage solution for creating dynamic, persistent volumes. 
+
+All data at rest on dbt Cloud servers is protected using AES-256 encryption. 
```
just double checking if it's meant to say 'at rest'?

matthewshaver replied:
Yup! When the data is on our servers (vs in transit over the internet), it is considered "at rest." It's a different type of encryption than when the data is moving over the internet

mirnawong1 commented on website/docs/docs/cloud/about-cloud/architecture.md:
```diff
@@ -0,0 +1,68 @@
+
+<img src="/img/docs/dbt-cloud/on-premises/data-flows.png" />
+
+For a more detailed breakdown of the dbt Cloud apps, [download the advanced architecture guide PDF](https://drive.google.com/uc?export=download&id=1lktNuMZybXfqFtr24J8zAssEfoL9r51S).
```
```suggestion
For a more detailed breakdown of the dbt Cloud application, [download the advanced architecture guide PDF](https://drive.google.com/uc?export=download&id=1lktNuMZybXfqFtr24J8zAssEfoL9r51S).
```

matthewshaver replied:
This should say "apps" plural. While the Cloud product can be called an application this is referencing individual apps all doing different things on their own and together behind the scenes.

mirnawong1 commented on website/docs/docs/cloud/about-cloud/architecture.md:
```diff
@@ -0,0 +1,68 @@
+
+dbt Cloud can communicate with several external services, including data platforms, git repositories, authentication services, and directories. All communications occur over HTTPS (attempts to connect via HTTP are redirected to HTTPS). dbt Cloud encrypts in transit using the TLS 1.2 cryptographic protocol. 
+
+TLS (Transport Layer Security) 1.2 is an industry-standard protocol for encrypting sensitive data while it travels over the public internet (which does not offer native encryption).
```
love this explanation

mirnawong1 commented on website/docs/docs/cloud/about-cloud/architecture.md:
```diff
@@ -0,0 +1,68 @@
+
+TLS (Transport Layer Security) 1.2 is an industry-standard protocol for encrypting sensitive data while it travels over the public internet (which does not offer native encryption).
+
+A typical scenario that might be seen frequently is an employee working in a public space, such as an airport or café. The user might be connected to an unsecured public network offered by a facility to which many others are also connected. What if there is a bad actor amongst them running a program (such as Wireshark) that can "capture" network packets and analyze them over the air?
```
great example! only small question of whether we want to explicitly use the name 'wireshark' and associate with bad actor? not sure if I'm being overly cautious though!

matthewshaver replied:
Good call out, but yes, Wireshark is a tool common to professionals and hackers alike. It's the most common tool used to monitor internet traffic over the air, capture network packets, and read unencrypted traffic. They're well aware of their status as a tool used for both good and evil.

mirnawong1 commented on website/docs/docs/cloud/about-cloud/architecture.md:
```diff
@@ -0,0 +1,68 @@
+
+The default settings of dbt Cloud enable local users with credentials stored in dbt Cloud. Still, integrations with various authentication services are offered as an alternative, including [single sign-on services](/docs/cloud/manage-access/sso-overview). Access to features can be granted/restricted by role using [RBAC](/docs/cloud/manage-access/enterprise-permissions).
+
+SSO features are essential because they reduce the number of credentials a user must maintain. Users sign in once and the authentication token is shared among integrated services (such as dbt Cloud). The token expires and must be refreshed at predetermined intervals, requiring the user to go through the authentication process again. If the user is disabled in the SSO provider service, their acces ...
```
```suggestion
SSO features are essential because they reduce the number of credentials a user must maintain. Users sign in once and the authentication token is shared among integrated services (such as dbt Cloud). The token expires and must be refreshed at predetermined intervals, requiring the user to go through the authentication process again. If the user is disabled in the SSO provider service, their access to dbt Cloud is disabled, and they cannot override this with local auth credentials. 
```

nghi-ly commented on _redirects:
```diff
@@ -653,8 +653,50 @@ https://tutorial.getdbt.com/* https://docs.getdbt.com/:splat 301!
 
+/website/docs/docs/contributing/documenting-a-new-adapter.md /guides/dbt-ecosystem/adapter-development/5-documenting-a-new-adapter 301
+
+# New Cloud directory redirects
```
just a flag in case it's helpful. please feel free ignore if it's not! 

i generally add redirects to the top of the file to avoid any rule ordering issues: https://docs.netlify.com/routing/redirects/#rule-processing-order

matthewshaver replied:
Moved them up! Thank you for the knowledge bombs

nghi-ly commented on website/docs/docs/cloud/manage-access/cloud-seats-and-users.md:
```diff
@@ -1,16 +1,18 @@
 ---
-title: "Seats and Users"
+title: "Users and Licenses"
```
minor nit. sentence case (vs title case)

```suggestion
title: "Users and licenses"
```

nghi-ly commented on website/docs/docs/cloud/manage-access/cloud-seats-and-users.md:
```diff
@@ -1,16 +1,18 @@
-title: "Seats and Users"
+title: "Users and Licenses"
 id: "seats-and-users"
+sidebar: "Users and Licenses"
```
```suggestion
sidebar: "Users and licenses"
```

nghi-ly commented on website/sidebars.js:
```diff
@@ -31,61 +42,112 @@ const sidebarSettings = {
-            },
+            }, // PrivateLink
+            "docs/cloud/develop-in-the-cloud",
+            "docs/cloud/dbt-cloud-tips",
```
ooc, should these two Cloud pages be under the new "About dbt Cloud" top-level?

matthewshaver replied:
I split the "develop in the cloud" into two docs so there is an "About Cloud IDE" section in the "About Cloud" folder but the dbt cloud tips is a bit of a wildcard. It's functional, so I'm not sure it belongs in the About Cloud folder but I'm open to ideas if this doesn't seem like the right place

nghi-ly commented on website/sidebars.js:
```diff
@@ -31,61 +42,112 @@ const sidebarSettings = {
       collapsed: true,
       items: [
         {
           type: "category",
```
does it make sense to include an "About setup" page ?

matthewshaver replied:
Written and submitting for review!

nghi-ly reviewed (commented):
looking good! lots of good work here! thank you! left some minor nits.

matthewshaver commented on website/docs/docs/cloud/about-cloud/architecture.md:
```diff
@@ -0,0 +1,68 @@
+
+TLS (Transport Layer Security) 1.2 is an industry-standard protocol for encrypting sensitive data while it travels over the public internet (which does not offer native encryption).
+
+A typical scenario that might be seen frequently is an employee working in a public space, such as an airport or café. The user might be connected to an unsecured public network offered by a facility to which many others are also connected. What if there is a bad actor amongst them running a program (such as Wireshark) that can "capture" network packets and analyze them over the air?
```
```suggestion
A typical scenario that might be seen frequently is an employee working in a public space, such as an airport or café. The user might be connected to an unsecured public network offered by a facility to which many others are also connected. What if there is a bad actor amongst them running a program that can "capture" network packets and analyze them over the air?
```

matthewshaver committed:
Apply suggestions from code review

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>
Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

matthewshaver committed:
Update _redirects

matthewshaver committed:
Merge branch 'current' into next

matthewshaver committed:
Merge branch 'next' of https://github.com/dbt-labs/docs.getdbt.com into next

runleonarun committed:
Merge branch 'next' into mwong-ide-tweaks

matthewshaver committed:
About page additions

matthewshaver committed:
Merge branch 'current' into next

matthewshaver committed:
Update website/docs/docs/cloud/about-cloud/architecture.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

matthewshaver committed:
Adding logos to setup

matthewshaver committed:
Merge branch 'next' of https://github.com/dbt-labs/docs.getdbt.com into next

mirnawong1 committed:
Merge branch 'next' into mwong-ide-tweaks

mirnawong1 committed:
Update develop-in-the-cloud.md (#2988)

updating ide docs to include some features that affect development. see
[slack
thread](https://dbt-labs.slack.com/archives/C043H8Z0P7H/p1678724314676849?thread_ts=1678122006.588049&cid=C043H8Z0P7H)

added:
- individual git commit/revert files
- yaml autocomplete
- 2+models

## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If linked to an open
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs,
see the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
<!--
Uncomment if you're publishing docs for a prerelease version of dbt
(delete if not applicable):
- [ ] Add versioning components, as described in [Versioning
Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
-->
- [ ] Review the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
and [About
versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this
PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link
testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally)
to update the links that point to the deleted page

matthewshaver committed:
Merge branch 'current' into next

mirnawong1 commented on website/docs/docs/cloud/about-cloud/architecture.md:
```diff
@@ -0,0 +1,68 @@
+
+The dbt Cloud application has two types of components: static and dynamic. The static components are always running to serve highly available dbt Cloud functions, like the dbt Cloud web application. On the other hand, the dynamic components are created ad-hoc to handle tasks such as background jobs or requests to use the IDE.
+
+dbt Cloud is available in most regions around the world in both [single tenant](/docs/cloud/about-cloud/tenancy#single-tenant)(AWS and Azure) and [multi-tenant](/docs/cloud/about-cloud/tenancy#multi-tenant) configurations.  
```
```suggestion
dbt Cloud is available in most regions around the world in both [single tenant](/docs/cloud/about-cloud/tenancy#single-tenant) (AWS and Azure) and [multi-tenant](/docs/cloud/about-cloud/tenancy#multi-tenant) configurations.  
```

mirnawong1 commented on website/docs/docs/cloud/about-cloud/tenancy.md:
```diff
@@ -0,0 +1,27 @@
+
+The Multi Tenant (SaaS) deployment environment refers to the SaaS dbt Cloud application hosted by dbt Labs. This is the most commonly used deployment and is completely managed and maintained by dbt Labs, the makers of dbt. As a SaaS product, a user can quickly [create an account](https://www.getdbt.com/signup/) on our North American servers and get started using the dbt and related services immed ...
+
+For more information about the dbt Cloud Production deployment see the below.
```
is this sentence still needed? seems like there isn't info re: prod deployment?

matthewshaver replied:
Great catch. I totally missed that. Removed it!

mirnawong1 commented on website/docs/docs/cloud/about-cloud/single-tenant.md:
```diff
@@ -13,5 +13,5 @@ _To learn more about setting up a dbt Cloud Single Tenant deployment, [please co
 
 For more information about the dbt Cloud Single Tenant deployment see the below.
```
is this sentence and the links below it still needed? or should it just link to dbt architecture?

matthewshaver replied:
Removed the entire page. Thanks for catching!

mirnawong1 commented on website/docs/docs/cloud/about-cloud/regions-ip-addresses.md:
```diff
@@ -17,4 +17,4 @@ dbt Cloud is [hosted](/docs/deploy/architecture) in multiple regions and will al
 
 
-[^1]: These regions support [multi-tenant](/docs/deploy/multi-tenant) deployment environments hosted by dbt Labs.
+[^1]: These regions support [multi-tenant](/docs/cloud/about-cloud/multi-tenant) deployment environments hosted by dbt Labs.
```
```suggestion
[^1]: These regions support [multi-tenant](/docs/cloud/about-cloud/tenancy) deployment environments hosted by dbt Labs.
```

mirnawong1 commented on website/docs/docs/about-setup.md:
```diff
@@ -0,0 +1,32 @@
+
+**dbt Core** is an open-source command line tool that can be installed locally in your environment, and communication with databases is facilitated through adapters.
+
+If you're not sure which is the right solution for you, read our [What is dbt?](/docs/introduction) and our [dbt Cloud Features](/docs/cloud/about-cloud/dbt-cloud-features) articles to help you decide. If you still have questions, don't hesitate to [contact us](https://www.getdbt.com/contact/).
```
```suggestion
If you're not sure which is the right solution for you, read our [What is dbt?](/docs/introduction) and our [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features) articles to help you decide. If you still have questions, don't hesitate to [contact us](https://www.getdbt.com/contact/).
```

mirnawong1 committed:
Update dbt-cloud-features.md

tweak env link

matthewshaver committed:
Update website/docs/docs/about-setup.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

matthewshaver committed:
Update website/docs/docs/cloud/about-cloud/architecture.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

matthewshaver committed:
Update website/docs/docs/cloud/about-cloud/regions-ip-addresses.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

matthewshaver committed:
Removing old tenant pages

matthewshaver committed:
remove errant line

matthewshaver committed:
Merge branch 'current' into next

mirnawong1 reviewed (approved):
amazing, so beautiful, and thank you for doing this and for our users!
